### ISSUE TITLE
How to unlock all steps?

### ISSUE THREAD

**Issue by joannesalfa** [cristijora/vue-form-wizard#113]
I need to know what's configuration to unlock all steps which is needed to edit the existing object.

Thanks.

**Comment by cristijora** [comment]
Hi @joannesalfa You could do it via refs 
There is no dedicated method for this yet
```js
  var wizard = this.$refs.wizard
   wizard.maxStep = wizard.tabs.length - 1
   wizard.tabs.forEach((tab)=>{
    tab.checked = true
   })
```
https://jsfiddle.net/bt5dhqtf/1594/

If you want, feel free to submit a PR with a method which does this or perhaps a prop to unlock steps

**Comment by cristijora** [comment]
Released [0.8.1](https://github.com/cristijora/vue-form-wizard/releases/tag/v0.8.1) which includes a method to activate all steps.
https://jsfiddle.net/bt5dhqtf/1598/

**Comment by joannesalfa** [comment]
Awesome, thank you!
